README.md

```diff
@@ -21,7 +21,8 @@
 </div>  
 <br>
 
-This project is an R interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library.
+This R package is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
+It enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions.
```

Comment:
Using something closer to the repository description

```suggestion
A R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library binds to R thanks to [htmlwidgets](http://www.htmlwidgets.org/).
```

Reply:
```suggestion
This R package enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions, with [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
Using JavaScript in R is handled by [htmlwidgets](http://www.htmlwidgets.org/) framework.
```

Reply:
Final proposal agreed with @csouchet 

```suggestion
This project is an R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is made possible by [htmlwidgets](http://www.htmlwidgets.org/), which provides an easy-to-use framework for bringing together R and the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library.
```